contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -2651,6 +2663,37 @@ exec_stmt_usedb_explain(PLtsql_execstate *estate, PLtsql_stmt_usedb *stmt, bool
 	return PLTSQL_RC_OK;
 }
 
+static int
+exec_stmt_grantdb(PLtsql_execstate *estate, PLtsql_stmt_grantdb *stmt)
+{
+	char 	*dbname = get_cur_db_name();
+	char	*dbowner = get_owner_of_db(dbname);
+	char	*login = GetUserNameFromId(GetSessionUserId(), false);	
+	ListCell *lc;
+	if (strcmp(login, dbowner) != 0)
```

Comment:
Please add a check for sysadmin and sysadmin members

Reply:
Added a check for the condition where the login is a member of "sysadmin" and relevant tests.